### PR TITLE
Updated readme files with URL new links 3.11

### LINKED ISSUE #343: Change the readme files in order to follow the documentation guide
Hello team

  According to PR https://github.com/wazuh/wazuh-documentation/pull/1907, in order to avoid duplicate work, we need to change our README files in all packages. The new content will redirect the users to the official documentation and now, all changes done in script uses etc has to be reflected in the documentation. 

Best regards,
Alberto R

### PR DESCRIPTION
**Pull request by alguacilaguamara**
Hello team.

This PR closes #343, I have changed the documentation of the package generation scripts to our new Wazuh 3.11 links with building instructions. 

Best regards,
Alejandro Alguacil.

### CODE REVIEW

File: aix/README.md [hunk 1]
```diff
@@ -5,39 +5,7 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
...
-        `# ./generate_wazuh_packages.sh -e`
-    * To build a wazuh-agent package for version 3.8.2, revision 3821 and store it in `/tmp`:
-
-        `# ./generate_wazuh_packages.sh -b v3.8.2 -s /tmp -a x86_64 -r 3821`.
-    * To install the dependencies, build a wazuh-agent package for version 3.8.2, revision 3821 and store it in `/tmp`:
-
-        `# ./generate_wazuh_packages.sh -b v3.8.2 -s /tmp -a x86_64 -r 3821`.
-
-3. When the execution finishes, you can find your `.ppc.rpm` package in specified folder.
+All info is in [Generate-aix-package.](https://documentation.wazuh.com/3.11/packaging/generate-aix-package.html)
```
Comment (alberpilot) [inline aix/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full AIX packages building documentation:"

File: debs/README.md [hunk 2]
```diff
@@ -3,47 +3,9 @@ Wazuh DEB packages
...
-    * To build a wazuh-manager package for amd64 (x86_64) and store it in `/tmp`:
-        `# ./generate_debian_package.sh -b 3.9 -s /tmp -t manager -a amd64 -r 0`.
-    * To build a wazuh-agent package in `/home/user` for i386 with release 2:
-        `# ./generate_debian_package.sh -b v3.8.2 -s /home/user -t agent -a i386 -r 2`.
-    * To build a wazuh-api package from branch 3.9 and store it in `/tmp`:
-        `# ./generate_debian_package.sh -b 3.9 -s /tmp -t api -a amd64 -r 0`.
-    * To build a wazuh-manager amd64 (x86_64) package for `/opt/ossec` directory and store it in `/tmp`:
-        `# ./generate_debian_package.sh -b 3.9 -s /tmp -t manager -a amd64 -r 0 -p /opt/ossec`.
-3. When the execution finishes, you can find your `.deb` packages in specified folder.
+All info is in [Generate-deb-package.](https://documentation.wazuh.com/3.11/packaging/generate-deb-package.html)
```
Comment (alberpilot) [inline debs/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full DEB packages building documentation:"

File: hpux/README.md [hunk 3]
```diff
@@ -3,50 +3,10 @@ Wazuh HP-UX packages
...
-    ```
-    * To install the needed dependencies:
-        `# ./generate_wazuh_packages.sh -u`.
-    * To download the sources from tag v3.9.0:
-        `# ./generate_wazuh_packages.sh -d v3.9.0`.
-    * To build a wazuh-agent package from the downloaded v3.9.0 sources:
-        `# ./generate_wazuh_packages.sh -b v3.9.0`.
-
-3. When the execution finishes, you can find your `tar` in the same directory where the sources are.
+All info is in [Generate-hpux-package.](https://documentation.wazuh.com/3.11/packaging/generate-hpux-package.html)
```
Comment (alberpilot) [inline hpux/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full HPUX packages building documentation:"

File: macos/README.md [hunk 4]
```diff
@@ -3,117 +3,10 @@ Wazuh macOS packages
...
-    ```
-    $ sudo xcode-select -r
-    ```
-    If this doesn't solve the issue, you will need to specify the path where Xcode is installed or unpacked:
-    ```
-    $ sudo xcode-select -s /path/to/Xcode.app
-    ```
-- `errSecInternalComponent` when running `codesign`: check the status of the login keychain. To solve it, you will need to close all the keychains and then run again the script.
-- `error: The specified item could not be found in the keychain`: this error may appear if `codesign` or `productsign` can't access to the Certificates, the private key or both. Check in the Keychain of your Mac hosts if they can be read by `codesign` and `productsign`.
+All info is in [Generate-osx-package.](https://documentation.wazuh.com/3.11/packaging/generate-osx-package.html)
```
Comment (alberpilot) [inline macos/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full macOS packages building documentation:"

File: ova/README.md [hunk 5]
```diff
@@ -3,37 +3,10 @@ Wazuh Virtual Machine
...
-      -h, --help             [  Util  ] Show this help.
-```
-
-To build an OVA with version 3.9.5 using elastic 7.3.0 and the stable repositories you can use:
-
-`# ./generate_ova.sh -b -v 3.9.5 -e 7.3.0 -r stable`
-
-   * **Stable:** The OVA uses released packages.
-   * **Unstable:** The OVA uses unstable packages.
+All info is in [Generate-ova.](https://documentation.wazuh.com/3.11/packaging/generate-ova.html)
```
Comment (alberpilot) [inline ova/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full OVA building documentation:"

File: rpms/README.md [hunk 6]
```diff
@@ -3,51 +3,9 @@ Wazuh RPM packages
...
-        `# ./generate_rpm_package.sh -b v3.8.2 -s /tmp -t agent -a x86_64 -j 15 -r 3`.
-    * To build a wazuh-api package from branch 3.9 and store it in `/tmp`:
-        `# ./generate_rpm_package.sh -b 3.9 -s /tmp -t api -a x86_64 -r 0`.
-    * To build a wazuh-manager x86_64 package for `/opt/ossec` directory and store it in `/tmp`:
-        `# ./generate_rpm_package.sh -b v3.8.2 -s /tmp -t manager -a x86_64 -r 0.1 -p /opt`.
-
-    If you build a package using `-d` parameter, you need to install the `wazuh-xxxxx` and the `wazuh-xxxxx-debuginfo` package in order to install the debugging symbols of the package if you want to debug the binaries using `gdb` for example.
-
-3. When the execution finishes, you can find your `.src.rpm` and the `.rpm` packages in specified folder.
+All info is in [Generate-rpm-package.](https://documentation.wazuh.com/3.11/packaging/generate-rpm-package.html)
```
Comment (alberpilot) [inline rpms/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full RPM packages building documentation:"

File: aix/README.md [hunk 7]
```diff
@@ -5,39 +5,7 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 
 ## Building RPM packages
```
Comment (alberpilot) [inline aix/README.md]: RPM Packages? Or AIX packages?

File: solaris/README.md [hunk 8]
```diff
@@ -3,11 +3,14 @@ Solaris packages
...
 
-- [Solaris 10](/solaris/solaris10/README.md)
-- [Solaris 11](/solaris/solaris11/README.md)
 - [Automated Solaris package generation using vagrant](/solaris/package_generation/README.md)
 - [Vagrantbox using packer](/solaris/packer/README.md)
 
+
+## Building Solaris packages
+
+All info is in [Generate-sol-package.](https://documentation.wazuh.com/3.11/packaging/generate-sol-package.html)
```
Comment (alberpilot) [inline solaris/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full Solaris packages building documentation:"

File: splunkapp/README.md [hunk 9]
```diff
@@ -3,43 +3,9 @@ Wazuh Splunk App package
...
-    * To build the app package for version 3.8.2, using Splunk with version 7.2.3, revision myrev and store it in `/splunk-app` you can either use:
-
-            # ./generate_wazuh_splunk_app.sh -b v3.8.1-7.2.3 -s /splunk-app -r myrev.
-            # ./generate_wazuh_splunk_app.sh -b 3.8.2 -s /splunk-app -r no -sp 7.2.3
-    * If no Splunk version is indicated the package generated will contain the latest version of the app for a given branch.
-
-            # ./generate_wazuh_splunk_app.sh -b 3.8 -s /splunk-app -r myrev
-
-3. Once the execution finishes, you will find your `.taz.gz` package in the specified folder or in /tmp/splunk-app if no folder was specified.
+All info is in [Generate-wazuh-splunk-app.](https://documentation.wazuh.com/3.11/packaging/generate-wazuh-splunk-app.html)
```
Comment (alberpilot) [inline splunkapp/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full Splunk App packages building documentation:"

File: wazuhapp/README.md [hunk 10]
```diff
@@ -3,39 +3,9 @@ Wazuh Kibana App package
...
-        -c, --checksum <path>     [Optional] Generate checksum
-        -h, --help                Show this help.
-
-    ```
-    * To build the app package from branch 3.8-6.7, revision myrev and store it in `/wazuh-app` you can use:
-
-            # ./generate_wazuh_app.sh -b 3.8-6.7 -s /wazuh-app -r myrev
-
-3. Once the execution finishes, you will find your `.zip` package in the specified folder or in `/tmp/wazuh-app` if no folder was specified.
+All info is in [Generate-wazuh-kibana-app.](https://documentation.wazuh.com/3.11/packaging/generate-wazuh-kibana-app.html)
```
Comment (alberpilot) [inline wazuhapp/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full Wazuh Kibana plugin packages building documentation:"

File: wpk/README.md [hunk 11]
```diff
@@ -3,68 +3,10 @@ WPK package
...
-        # ./generate_wpk_package.sh -t linux -b 3.8 -d /home/wpk -k /tmp/keys -o linux-3_8.wpk
-        ```
-    For windows packages it is necessary to have a msi package, you can find the latest packages here: [wazuh packages](https://documentation.wazuh.com/current/installation-guide/packages-list/index.html#packages)
-
-    * To build the WPK package for windows called windows-3_9_0.wpk from tag v3.9.0 and store it in /home/wpk, while having the keys stored in /tmp/keys and the msi stored in /tmp/msi/wazuh-agent-3.9.0-1.msi
-        ```shellsession
-        # /generate_wpk_package.sh -t windows -b v3.9.0 -d home/wpk -k /tmp/keys -o windows-3_9_0.wpk -pd /tmp/msi/wazuh-agent-3.9.0-1.msi
-        ```
-    3. Once the execution finishes, you will find your `.zip` package in the specified folder.
+All info is in [Generate-wpk-package.](https://documentation.wazuh.com/3.11/packaging/generate-wpk-package.html)
```
Comment (alberpilot) [inline wpk/README.md]: Please, replace "All info is in" by "Please, visit the following link for the full WPK packages building documentation:"

File: aix/README.md (hunk 1 above)
Comment (alguacilaguamara) [reply aix/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: debs/README.md (hunk 2 above)
Comment (alguacilaguamara) [reply debs/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: hpux/README.md (hunk 3 above)
Comment (alguacilaguamara) [reply hpux/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: macos/README.md (hunk 4 above)
Comment (alguacilaguamara) [reply macos/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: ova/README.md (hunk 5 above)
Comment (alguacilaguamara) [reply ova/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: rpms/README.md (hunk 6 above)
Comment (alguacilaguamara) [reply rpms/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: aix/README.md (hunk 7 above)
Comment (alguacilaguamara) [reply aix/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: solaris/README.md (hunk 8 above)
Comment (alguacilaguamara) [reply solaris/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: splunkapp/README.md (hunk 9 above)
Comment (alguacilaguamara) [reply splunkapp/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: wazuhapp/README.md (hunk 10 above)
Comment (alguacilaguamara) [reply wazuhapp/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

File: wpk/README.md (hunk 11 above)
Comment (alguacilaguamara) [reply wpk/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.

**Pull request merged**